docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -169,14 +185,41 @@ For a server name to be included, it must match the regular expression specified
 
 ### service block
 
-Name           | Type     | Description                                           | Default | Required
----------------|----------|-------------------------------------------------------|---------|---------
-`use_api`      | `string` | Use API calls to collect service data instead of WMI. | `false` | no
-`where_clause` | `string` | WQL 'where' clause to use in WMI metrics query.       | `""`    | no
+Name                  | Type     | Description                                           | Default | Required
+----------------------|----------|-------------------------------------------------------|---------|---------
+`enable_v2_collector` | `string` | Enable V2 service collector.                          | `""`    | no
+`use_api`             | `string` | Use API calls to collect service data instead of WMI. | `false` | no
+`where_clause`        | `string` | WQL 'where' clause to use in WMI metrics query.       | `""`    | no
 
 The `where_clause` argument can be used to limit the response to the services you specify, reducing the size of the response.
 If `use_api` is enabled, 'where_clause' won't be effective.
 
+The v2 collector can query service states much more efficiently, but can't provide general service information.
```

Comment:
Can we provide something around what "general service information is"?

Reply:
v2 can only provide the service_state metric, while v1 can provide much more information like status, start_mode, info.

https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.service.md

Background: There is a Microsoft API to requests the state from all service through one API request. That avoids loops through each single service and make the collector extremely fast.

Other metric will be re-introduced via a separate collector, e.g. `service_info` collector.